Functions.Templates/ProjectTemplate/CSharp-Isolated/.template.config/template.json

```diff
@@ -26,7 +26,7 @@
         "AzureFunctionsVersion": {
             "description": "The setting that determines the target release",
             "type": "parameter",
-            "defaultValue": "V3",
+            "defaultValue": "V4",
```

Comment:
@soninaren I'm surprised this V3 here. Thoughts?

Reply:
That is surprising here. Though vs relies on another property generated by net sdk pacakge.